src/pages/FileFields.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { FC, useEffect, useState } from 'react';
+import { emptyFile, File, Uppy } from '../types/file.types';
+import StatusBarComponent from '@uppy/react/src/StatusBar';
+import '@uppy/core/dist/style.css';
+import '@uppy/status-bar/dist/style.css';
+import styled from 'styled-components';
+import { Paper, Typography } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import placeholderImage from '../resources/images/placeholder.png';
+
+const StatusBarWrapper = styled.div`
+  width: 100%;
+`;
+
+const MainFileWrapper = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+`;
+
+const MainFileImageWrapper = styled.div`
+  margin-top: 1rem;
+  margin-right: 2rem;
+  max-height: 200px;
+  max-width: 200px;
+`;
+
+const MainFileMetadata = styled.div`
+  flex-grow: 1;
+`;
+
+interface FileFieldsProps {
+  uppy: Uppy;
+}
+
+const FileFields: FC<FileFieldsProps> = ({ uppy }) => {
+  const [uploadedFiles, setUploadedFiles] = useState<File[]>([]);
```

Comment:
trengs dette? opplastede filer vil uansett ligge i `uppy.getFiles()`

Reply:
dette er blåkopi fra NVA,da. Jeg kan teste med getFiles fra uppy

Reply:
funker fint det ja :-)

Reply:
men det er vel pga at man vil sette flere metadata på filene

Reply:
må ikke ta alt vi har gjort for god fisk heller :D